src/PrestaShopBundle/Controller/Admin/Improve/ModuleController.php

```diff
@@ -192,9 +192,9 @@ public function moduleAction(Request $request)
 
         $module = $request->get('module_name');
         $source = $request->query->get('source');
-        $moduleManager = $this->container->get('prestashop.module.manager');
```

Comment:
Is this still relevant? Or is it a modification from your previous implementation? If it's not required maybe we should revert this and keep the PR as small as possible to only have code related to what we're aiming here

Reply:
After discussion, the purpose here is to harmonize our controllers and only use `$this->get` A PHPStan rule should be added to prevent his kind of usage later